imageio/plugins/rawpy.py

```diff
@@ -49,10 +49,11 @@ def __init__(self, request: Request) -> None:
                     raise InitializationError(
                         f"RawPy can not read {request.raw_uri}."
                     ) from None
-        elif request.mode.io_mode == IOMode.write:
-            raise InitializationError(
-                "RawPy does not support writing."
-            ) from None
+        # Why do we need this when we are raising NotImplementedError in the write method ?
```

Comment:
Because the `InitializationError` is used during plugin selection. It allows ImageIO to try this plugin to fulfill a request (e.g. for an unknown file format), realize it won't work, and try a different one before returning control to the user.

Reply:
Done. Thanks for the review.